Camera2SlowMotion/app/src/main/java/com/example/android/camera2/slowmo/MainActivity.java

```diff
@@ -107,24 +107,24 @@ private void analyze(){
             imageHolder = InputImage.fromBitmap(frameList.get(i), 0);
             final int finalI = i;
             Task<Text> result =
-                recognizer.process(imageHolder)
-                    .addOnSuccessListener(new OnSuccessListener<Text>() {
-                        @Override
-                        public void onSuccess(Text visionText) {
-                            // Task completed successfully
-                            results.add(visionText.getText());
-                            Log.d("Rokus logs", "Text detected at index:" + finalI + " " + visionText.getText());
-                        }
-                    })
-                    .addOnFailureListener(
-                        new OnFailureListener() {
-                            @Override
-                            public void onFailure(@NonNull Exception e) {
-                                // Task failed with an exception
-                                e.printStackTrace();
-                                e.getCause();
-                            }
-                        });
+                    recognizer.process(imageHolder)
+                            .addOnSuccessListener(new OnSuccessListener<Text>() {
+                                @Override
+                                public void onSuccess(Text visionText) {
+                                    // Task completed successfully
+                                    results.add(visionText.getText());
+                                    Log.d("Rokus logs", "Text detected at index:" + finalI + " " + visionText.getText());
+                                }
+                            })
+                            .addOnFailureListener(
+                                    new OnFailureListener() {
+                                        @Override
+                                        public void onFailure(@NonNull Exception e) {
+                                            // Task failed with an exception
+                                            e.printStackTrace();
+                                            e.getCause();
```

Comment:
nit: I don't think this file should be changed in 6da0c5 (since it is the import of sample code), if you must change the indentation maybe do it in the later commit (or fix the earlier commit). Also, I think the comment about e.getCause() is still outstanding.